plugins/main/windows/l2bridge/README.md

```diff
@@ -0,0 +1,28 @@
+# L2Bridge  plugin (Windows)
+
+## Overview
+
+With L2bridge plugin, all containers (on the same host) are plugged into an L2Bridge network that has one endpoint in the host namespace.
+
+
+## Example configuration
+```
+{
+	"name": "mynet",
+	"type": "win-l2bridge",
```

Comment:
Need to make the type consistent with the actual plugin name, eg when renamed to "win-bridge" need to make this "type": "win-bridge",